nepalingo-web/package-lock.json

```diff
@@ -0,0 +1,6876 @@
+{
+  "name": "nepalingo-web",
+  "version": "0.0.0",
+  "lockfileVersion": 3,
```

Comment:
It looks like there was some issue with your pnpm version - can you make sure to update to the latest pnpm and then rerun install?